src/System.Text.Json/tests/JsonElementWithNodeParentTests.cs

```diff
@@ -122,7 +122,7 @@ public static void TestString()
         [Fact]
         public static void TestBytesFromBase64()
         {
-            Assert.Throws<NotSupportedException>(() => new JsonString().AsJsonElement().GetBytesFromBase64());
+            Assert.Equal(Encoding.UTF8.GetBytes("value"), new JsonString("dmFsdWU=").AsJsonElement().GetBytesFromBase64());
```

Comment:
This needs a lot more tests... whitespace-containing, two trailing =, zero trailing =, something that returns false (including zero, one, and two trailing = in the false-returning inputs)